src/handleFormSubmit.js

```diff
@@ -3,7 +3,7 @@ import backgroundScript from "./backgroundScript";
 
 const handleFormSubmit = async (input, amountFound) => {
   let [tab] = await chrome.tabs.query({ active: true, currentWindow: true });
-  const selectors = input.value.split(" ");
+  const selectors = input.value ? input.value.split(" ") : [];
```

Comment:
If you are checking if the value is there, maybe also check if the value is splittable? if we put number as a input it will crash on split :/

Reply:
will do

Reply:
It will fail because it is not a valid selector, but the input.value is a string so whatever is there can be splitted. Instead I wrapped the place where the selector is used in trycatch to catch the selectors that are not valid